zzz_generated/bridge-app/zap-generated/CHIPClientCallbacks.h

```diff
@@ -30,15 +30,6 @@
 #include <lib/support/Span.h>
 
 // List specific responses
-typedef void (*BindingBindingListAttributeCallback)(
-    void * context,
-    const chip::app::DataModel::DecodableList<chip::app::Clusters::Binding::Structs::TargetStruct::DecodableType> & data);
-typedef void (*BindingGeneratedCommandListListAttributeCallback)(void * context,
-                                                                 const chip::app::DataModel::DecodableList<chip::CommandId> & data);
-typedef void (*BindingAcceptedCommandListListAttributeCallback)(void * context,
-                                                                const chip::app::DataModel::DecodableList<chip::CommandId> & data);
-typedef void (*BindingAttributeListListAttributeCallback)(void * context,
-                                                          const chip::app::DataModel::DecodableList<chip::AttributeId> & data);
```

Comment:
Binding's server side cluster is not enabled for the app. Therefore the server side attributes are not generated either.